pkg/operator/ceph/pool/controller.go

```diff
@@ -350,9 +350,12 @@ func (r *ReconcileCephBlockPool) reconcile(request reconcile.Request) (reconcile
 			if r.blockPoolContexts[blockPoolChannelKey].started {
 				logger.Debug("pool monitoring go routine already running!")
```

Comment:
For a follow-up PR: it would be helpful for debugging if we (1) add `blockPoolChannelKey` to the output here, and (2) debug log in the `else` below when we are starting the channel checker.